### DISCUSSION TITLE
New install/ Error Messages/ No camera visible

### DISCUSSION THREAD

**Discussion by Joebus1** [aaronwmorris/indi-allsky#1664] [Q&A] [answered]
I'm migrating over from my facebook post(https://www.facebook.com/photo?fbid=10228210211305612&set=gm.1615700035684208&idorvanity=172438633343696) now that I have found this forum. 

Since that post I have re-installed Bookworm lite 64bit, compiled indi, and installed indi-allsky.  The camera is a Rpi HQ now plugged in the correct way.

These are the errors I get when going to the Focus Tool with focus mode enabled.(Saved from config and restarted)
![image](https://github.com/user-attachments/assets/84b9d79f-5ffe-4a8d-9fde-b112e49a820f)
![image](https://github.com/user-attachments/assets/0a9caf8d-7c38-4f80-8c6e-44311a56eabd)

This is the real-time log snapshot, not sure where to find or how to get the full log txt.
![image](https://github.com/user-attachments/assets/8178f0e9-4771-443a-8a63-6f5b899ffb27)

This is my support info:

[allsky_dump.txt](https://github.com/user-attachments/files/17873910/allsky_dump.txt)

**Comment by aaronwmorris** [comment, marked as answer]
Okay, this was my fault.  I had done some recent updates to the python module requirements and rawpy did not get installed.

Merged #1666 to fix.  Pull down the latest code and re-run `setup.sh` and it should fix it for you.

**Reply by Joebus1** [reply]
Fixed, I got an image now!   Thank you and good find.  Now off to study all the options available :)
![image](https://github.com/user-attachments/assets/30f27e30-a79c-4813-a665-64884b15296b)
